doc/build-unix.md

```diff
@@ -61,34 +61,40 @@ Dependency Build Instructions: Ubuntu & Debian
 ----------------------------------------------
 Build requirements:
 
-	sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
+        sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
```

Comment:
Not going to worry about spaces too much here, sorry. Let's focus on technical content.